packages/flutter/lib/src/material/floating_action_button.dart

```diff
@@ -21,7 +22,8 @@ class FloatingActionButton extends StatefulComponent {
     this.backgroundColor,
     this.elevation: 6,
     this.highlightElevation: 12,
-    this.onPressed
+    this.onPressed,
+    this.mini: false
```

Comment:
Yup, just the two, regular and mini. It's just a little ways down on this page: https://www.google.com/design/spec/components/buttons-floating-action-button.html.